packages/ancient/ancient.0.9.1/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+authors: "Richard Jones et.al."
+homepage: "https://github.com/UnixJunkie/ocaml-ancient"
+bug-reports: "https://github.com/UnixJunkie/ocaml-ancient/issues"
+dev-repo: "git+https://github.com/UnixJunkie/ocaml-ancient.git"
+maintainer: "unixjunkie@sdf.org"
+build: [
+  ["sh" "-c" "cd mmalloc && ./configure"]
+  [make "depend"]
+  [make]
+]
+remove: ["ocamlfind" "remove" "ancient"]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "ocamlfind"
```

Comment:
```suggestion
  "ocamlfind" {build}
```
do you need ocamlfind as a runtime dependency? I couldn't find any use in the source code